roles/openmrs/tasks/main.yml

```diff
@@ -28,17 +28,6 @@
     group: root
     mode: 0700
 
-- name: Import the OpenMRS trigger .sql file
```

Comment:
Please don't remove this task.

Reply:
@keymane removed that one.

Reply:
Cool. @manutarus' fix for the issue was merged to master.